src/index.js

```diff
@@ -0,0 +1,6 @@
+const app = require('./app');
+
+const port = process.env.PORT || 5000;
+app.listen(port, () => {
+  console.log(`Listening: http://localhost:${port}`);
```

Comment:
Listening at port

Reply:
it will give u the url so Listening: http://localhost:${port}